.github/workflows/on_push_action.yml

```
@@ -5,14 +5,24 @@ on: [push,pull_request]
 jobs:
   ubuntu-build-and-test:
     runs-on: ubuntu-latest
-    container:
-      image: gnosispm/miplib-backends:1.1.0
     steps:
     - name: Checkout
       uses: actions/checkout@v2.0.0
-    - name: Build project
-      uses: ashutoshvarma/action-cmake-build@master
+    - name: Build Container
+      uses: docker/build-push-action@v2
       with:
-        build-dir: ${{ runner.workspace }}/build
-        build-type: Release
-        run-test: true
+        context: .
+        file: "${{ github.workspace }}/docker/Dockerfile"
+        push: false
+        tags: miplib:latest
+
+    - name: Run the build process with Docker
+      uses: addnab/docker-run-action@v3
+      with:
+        image: miplib:latest
+        options: -v ${{ github.workspace }}:/var/www
+        run: |
+          cd /var/www
```

```suggestion
        options: -v ${{ github.workspace }}:/src
        run: |
          cd /src
```

Feels weird to build `miplib` in `/var/www`

---

That's true. I just copied that from an example. And went to bed when I had something that worked. 😅